lib/ansible/modules/cloud/ovirt/ovirt_host_network.py

```diff
@@ -362,8 +367,12 @@ def main():
         nics_service = host_service.nics_service()
         nic = search_by_name(nics_service, nic_name)
 
+        if module.params["sync_networks"]:
```

Comment:
Please call this only when needed and report change=true/false if it was called and is needed.